### ISSUE TITLE
tags can't be searched (in tags selector config)

### ISSUE THREAD

**Issue by mrksjs** [zylklab/zk-dynamic-searches-dashlet#2]
Hi 

First of all, cool project! Share is lacking such dashlet/functionality.

__This concerns only Mode 0__

**My Issue/Change Request:**
The Tag selector does not allow searching - If you have more than 100 Tags (default share config), you are out of luck.

_Additionally_, the 'select Tags' dialog allows going up to the parent 'categories', if you go up again you will end up in the store root (afaik) with no option to get back to categories/Tags anymore. 

thanks & regards

markus

**Comment by CesarCapillas** [comment]
Hi Markus:

Thank you very much for the feedback. 
May you confirm us your Alfresco version and search subsystem ?

Regards.

--C.

P.S: Internally we use Alfresco 201602GA, with SOLR4 and more than 315 tags applied.

**Comment by mrksjs** [comment]
Hi Cesar

I am running 201702GA with solr4.
The Tag search itself is working of course, but the Tag-Selector in the configuration of the dashlet does not allow searching.


thanks & regards

markus

**Comment by CesarCapillas** [comment]
Yes, I can go upstairs with Tag Selector :) and I see that the Tag-Selector in update properties can search too, but not in the dashlet. I'll check it, and I'll keep you informed.

Best regards.
--C.

**Comment by CesarCapillas** [comment]
It seems that this behaviour is because same UI component is used both for categories and tags.
